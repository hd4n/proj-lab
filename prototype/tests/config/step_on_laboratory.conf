@fields
em e1
la l1

@neighbors
e1 l1

@virologists
v1 e1