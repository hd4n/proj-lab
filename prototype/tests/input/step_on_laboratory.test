load step_on_laboratory.conf
step v1 l1
nextround
getstats v1
getstats l1