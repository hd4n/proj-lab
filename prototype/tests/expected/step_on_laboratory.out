Load: sikeres
step: v1 l1
nextround: sikeres

stats: v1
	fi_l1
	di_l1
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_null

stats: l1
	ci_v1
	ne_e1
	co_null
